CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.0.0)
+cmake_minimum_required(VERSION 3.6.3)
```

Comment:
we can't accept this bump as this will block us from integrating with Genivi SDK (there're using 3.0)